Self-hosted: in-product purchase (#6136)
mattermost/docs: source/about/self-hosted-subscriptions.rst

When you subscribe to an annual plan with Mattermost, you agree to provide quarterly reports of the actual number of registered users within your system if you're above the amount you subscribed in your purchase order. A registered user is a user who has an account in a workspace and does not show as *Inactive* in **System Console > User Management > Users**.

We'll send you an email notice around the end of the quarter reminding you to send us your report. 

> From Mattermost v7.7, you can send us this information from **System Console > License and Edition > True-up Review**. 

Reviewer: Sorry for spotting this after the fact @justinegeffen , but a couple things:

- This feature is going to be ready for 7.9, not 7.7
- They can send the info via `System Console > Site Statistics` once prompted by their CSM

Author: Ok, so what's the process for 7.7? Still the same?

Author: I've removed "From Mattermost v7.7, you can send us this information from **System Console > License and Edition > Site Statistics**." - what is the process that's followed in 7.7? 

- Is there a bundle that can be generated?
- Or do users still need to follow the screenshot process for now?

> If your deployment doesn't have internet access, you'll need to export a bundle which is uploaded via the Customer Portal. It's then accessed by the Custome Success team. In **System Console > True-up Review**, select **Export Bundle**. Then log into your Customer Portal account, navigate to the **True-up Review** section and upload the bundle.

Reviewer: @justinegeffen actually since we found out the effort here is considerable, we decided for v1 for customers to export the bundle and then send it to their CSM via email

Author: Updated, thanks!

> If your deployment doesn't have internet access, you'll need to export a bundle which is uploaded via the Customer Portal. It's then accessed by the Custome Success team. In **System Console > True-up Review**, select **Export Bundle**. Then log into your Customer Portal account, navigate to the **True-up Review** section and upload the bundle.

Author: ```suggestion
If your deployment doesn't have internet access, you'll need to export a bundle and email it to your Customer Success Manager. You can create the bundle in **System Console > True-up Review**, select **Export Bundle**.
```
